_sass/styles.scss

```diff
@@ -70,6 +70,10 @@
 // ==========================================================================
 @import "uswds";
 
-// -------------------------------------
-// Import theme custom styles
-// @import "uswds-theme-custom-styles";
+//
+// Custom styles
+// ==========================================================================
+@import "uswds-custom-styles";
+
+// Custom non-USWDS styles
+@import "custom-components";
```

Comment:
10. I created a new partial called `_custom-components.scss` and those (for now) are styles for non-USWDS components. You should split this file out into several if you find yourself with many custom components.